### DISCUSSION TITLE
n-select width (as display:inline-block)

### DISCUSSION THREAD

**Discussion by larryval** [tusen-ai/naive-ui#3075] [General]
Hello All!

Well , how can I have the n-select width component displays like native HTML <select></select> (I mean like display:inline-block , and the width of the component is adapted by the wider option).
I've tried
```js
:consistent-menu-width="false"
```
but with no result, the n-select comp takes the full width of it's parent.
thank's in advance if someone have the answer !

**Comment by XieZongChen** [comment]
Maybe we should provide a props that can be set... 🤔

**Reply by larryval** [reply]
well why not ? !

**Reply by 07akioni** [reply]
Only if we make placeholder DOM take space then we can make it displays as inline block.

**Reply by XieZongChen** [reply]
> Only if we make placeholder DOM take space then we can make it displays as inline block.

Yeah, has changed a lot.
